src/mixins/payment-mixin.ts

```diff
@@ -536,7 +536,8 @@ export class PaymentMixin extends Mixins(ActionMixin) {
       if (!paymentResponse) throw new Error('Got error from getNameRequestPayment()')
 
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      const { payment, sbcPayment = { receipts: [], status_code: '' }, statusCode, completionDate } = paymentResponse
+      // Note this was broken for a while. It was fixed by adding [0] to the response.
```

Comment:
I think we don't need this comment. However, a comment indicating that paymentResponse may be an object or an array or null might be useful (if true).

Reply:
I've changed it back since, it can affect too many pathways